msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -935,50 +931,36 @@ The following code provides an example of meeting start event payload:
 The following code provides an example of meeting end event payload:
 
 ```json
-{ 
-    "name": "application/vnd.microsoft.meetingEnd", 
-    "type": "event", 
-    "timestamp": "2021-04-29T16:17:17.4388966Z", 
-    "id": "123", 
-    "channelId": "msteams", 
-    "serviceUrl": "https://microsoft.com", 
-    "from": { 
-        "id": "user id", 
-        "aadObjectId": "aadObjectId" 
-    }, 
-    "conversation": { 
-        "isGroup": true, 
-        "tenantId": "tenantId", 
-        "id": "thread id" 
-    }, 
-    "recipient": { 
-        "id": "user id", 
-        "name": "user name" 
-    }, 
-    "entities": [ 
-        { 
-            "locale": "en-US", 
-            "country": "US", 
-            "type": "clientInfo" 
-        } 
-    ], 
-    "channelData": { 
-        "tenant": { 
-            "id": "channel id" 
-        }, 
-        "source": null, 
-        "meeting": { 
-            "id": "meeting Id" 
-        } 
-    }, 
-    "value": { 
-        "MeetingType": "Scheduled", 
-        "Title": "Meeting Start/End Event in Canary", 
-        "Id": "19:meeting_NTM3ZDJjOTUtZGRhOS00MzYxLTk5NDAtMzY4M2IzZWFjZGE1@thread.v2", 
-        "JoinUrl": "url", 
-        "EndTime": "2021-04-29T16:17:17.4388966Z" 
-    }, 
-    "locale": "en-US" 
+{
+  "name": " application/vnd.microsoft.meetingEnd",
+  "type": "event",
+  "timestamp": "2023-02-23T19:34:07.478Z",
+  "localTimestamp": "2023-02-23T11:34:07.478-8",
+  "channelId": "msteams",
+  "serviceUrl": "https://smba.trafficmanager.net/teams/",
+  "from": {
+    "id": "user_id"
+  },
+  "conversation": {
+    "isGroup": true,
+    "conversationType": "groupchat",
+    "id": "conversation_id"
+  },
+  "recipient": {
+    "id": "28:f5d48856-5b42-41a0-8c3a-c5f944b679b0"
```

Comment:
same comment as above

Reply:
Hi @tacri ,

We have updated the id with a test tenant bot id (28:65f50003-e15d-434a-9e14-0fcfeb3d7817). Can you please confirm.

thanks,
Yash